core/src/main/java/org/apache/iceberg/ManifestFilterManager.java

```diff
@@ -71,9 +70,9 @@ public String partition() {
   private final PartitionSet deleteFilePartitions;
   private final PartitionSet dropPartitions;
   private final CharSequenceSet deletePaths = CharSequenceSet.empty();
+  private final Set<F> deleteFiles = newFileSet();
```

Comment:
Can we co-locate `deleteFiles` and `deleteFilePartitions` as `deleteFilePartitions` essentially contains a set of partitions derived from `deleteFiles`.